src/commentModal.js

```diff
@@ -0,0 +1,57 @@
+const commentModal = (food) => `
+    <div class="meal-details w-full mx-5 md:w-1/3 h-5/6 bg-white rounded-lg p-2 z-50 opacity-100 overflow-y-auto relative">
+      <!-- close btn -->
+      <button type="button" class="float-right mr-1 absolute right-0 top-0 sticky" id="close-modal">
+        <i class="fa fa-close text-2xl text-orange-500 hover:text-red-300"></i>
+      </button>
+
+      <!-- Meal contents -->
+      <div class="meal-details-content">
+
+        <div class="w-full pt-2 pb-4 px-5">
+          <div class="recipe-meal-image flex justify-center mt-2">
+            <img
+              src="${food.strMealThumb}"
+              alt="Meal name"
+              class="rounded-lg shadow-lg"
+              width="350"
+            />
+          </div>
+          <div class="mt-2">
+            <div class=" my-1">
+              <h3 class="text-center text-xl md:text-3xl text-orange-500 mt-5 mb-2">${food.strMeal}</h3>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] In the comment popup, you need to show details about the selected item like ingredients, or anything which you can fetch from the API.

Reply:
Ok @Dipeshtwis , We've successfully made all updates and now fetching more details for each food item, its ingredients, and description. Thanks for the review and for making our app looks more appealing when showing more details. 🤝